.travis.yml

```diff
@@ -4,6 +4,8 @@ sudo: required
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - php: hhvm   # ZIP Error(11): Status(0): Arhive(cache/zipTest.zip): Directory(cache)
```

Comment:
What was the reason for adding ```hhvm``` as an allowed failure? afaik we don't actually use ```hhvm``` in our Travis builds.